dbt/adapters/duckdb/plugins/glue.py

```diff
@@ -132,12 +133,64 @@ def _convert_columns(column_list: Sequence[Column]) -> Sequence["ColumnTypeDef"]
     return column_types
 
 
-def _create_table(client: "GlueClient", database: str, table_def: "TableInputTypeDef") -> None:
+def _create_table(
+    client: "GlueClient", database: str, table_def: "TableInputTypeDef", partition_columns
+) -> None:
     client.create_table(DatabaseName=database, TableInput=table_def)
+    # Create partition if relevant
+    if partition_columns != []:
+        partition_names = [column["Name"] for column in partition_columns]
+        partition_values = [column["Value"] for column in partition_columns]
 
+        partition_location = table_def["StorageDescriptor"]["Location"]
+        for name, value in zip(partition_names, partition_values):
+            partition_location += f"/{name}={value}"
```

Comment:
I have an aesthetic preference for something like:

```
partition_location = table_def["StorageDescriptor"]["Location"]
partition_components = [partition_location]
for c in partition_columns:
    partition_components.append("=".join((c["Name"], c["Value"]))
partition_location = "/".join(partition_components)
```

Reply:
Done